dotnet/src/IntegrationTests/Connectors/MistralAI/ChatCompletion/MistralAIChatCompletionTests.cs

```diff
@@ -163,7 +189,31 @@ public async Task ValidateGetChatMessageContentsWithAutoInvokeAsync()
     }
 
     [Fact] // (Skip = "This test is for manual verification.")
-    public async Task ValidateGetChatMessageContentsWithAutoInvokeAndInvocationFilterAsync()
+    public async Task ValidateGetChatMessageContentsWithNoFunctionsAsync()
+    {
+        // Arrange
+        var model = this._configuration["MistralAI:ChatModel"];
+        var apiKey = this._configuration["MistralAI:ApiKey"];
+        var service = new MistralAIChatCompletionService(model!, apiKey!);
+        var executionSettings = new MistralAIPromptExecutionSettings { ToolCallBehavior = MistralAIToolCallBehavior.NoKernelFunctions };
+        var kernel = new Kernel();
+        kernel.Plugins.AddFromType<WeatherPlugin>();
+
+        // Act
+        var chatHistory = new ChatHistory
+        {
+            new ChatMessageContent(AuthorRole.User, "What is the weather like in Paris?")
+        };
+        var response = await service.GetChatMessageContentsAsync(chatHistory, executionSettings, kernel);
+
+        // Assert
+        Assert.NotNull(response);
+        Assert.Single(response);
+        Assert.Contains("GetWeather", response[0].Content, System.StringComparison.Ordinal);
+    }
+
+    [Fact] // (Skip = "This test is for manual verification.")
```

Comment:
```suggestion
    [Fact]
```

Reply:
This is a reminder that these tests need to be skipped when merging to main